### DISCUSSION TITLE
LNK1104 in Visual Studio - can't open "\usr\local\lib\libTgBot.a"

### DISCUSSION THREAD

**Discussion by Amirhan-Taipovjan-Greatest-I** [reo7sp/tgbot-cpp#216] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51203385/181752344-8b3edffd-c85a-4337-a0a4-6b7ce987dce4.png)

I've installed tgbot-cpp for many times and I still get that error.

I have:
- Windows 10 Pro 21H2
- vcpkg with all installed packages like boost-system, openssl, zlib, curl and even tgbot-cpp
- CMake
- Visual Studio 2022 Community Edition
- Builded by CMake tgbot-cpp directories

I even tried to build and use the sample - "Echobot", but it's not working too!

**Comment by we-lu** [comment, marked as answer]
You need to update the CMakeLists. You need to change all the paths to where your tgbot library is located

**Reply by Amirhan-Taipovjan-Greatest-I** [reply]
Looks like I need to change "/usr/local/lib/libTgBot.a" to "C:/src/vcpkg/installed/x64-windows/lib/TgBot.lib" or something similar according to My own installation path, right?

**Reply by we-lu** [reply]
Yes, that should work
